covigator/accessor/ena_accessor.py

```diff
@@ -191,6 +195,12 @@ def _parse_ena_run(self, run):
     def _complies_with_inclusion_criteria(self, ena_run: dict):
         # NOTE: this uses the original dictionary instead of the parsed SampleEna class for performance reasons
         included = True
+        # Skip samples with empty collection date - Note this checks not if collection date is too early
+        if not self.disable_collection_date_filter:
```

Comment:
This is just a style comment that I normally try to avoid as for me it makes the code slightly more difficult to understand.

Double negations. When interpreting the code in your head and going through the logic, double negations are very error prone (for me, I did not run a trial on it). Here you do `not` and `disable` which means `enable`, so I would call the variable here `enable`. I understand that this comes from CLI where you want your default to be enabled and hende the flag is `disabled`, you can keep disable in the CLI.


You don't need to apply this change.